tests/dist/DistTestExecutor.h

```diff
@@ -27,6 +28,14 @@ class DistTestExecutor final : public faabric::scheduler::Executor
       int threadPoolIdx,
       int msgIdx,
       std::shared_ptr<faabric::BatchExecuteRequest> req) override;
+
+    faabric::util::SnapshotData snapshot() override;
+
+    uint8_t* snapshotMemory = nullptr;
+    size_t snapshotSize = 0;
```

Comment:
Originally I hadn't set defaults on the `snapshotMemory` and `snapshotSize`, which worked on my machine as the size was getting set to zero, but in Github Actions it was getting set to a non-zero value which was breaking everything. The take-away is that we must always set default values on class members. I looked for a clang-tidy rule to enforce this but couldn't find it.